codegen/smithy-ruby-codegen-test/integration-specs/relative_middleware_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module WhiteLabel
+  describe Client do
+    context 'relative middleware ordering' do
```

Comment:
I do wish we had a better way to link the various parts of these tests together better.... but haven't been able to think of one.

I think maybe we should at least add a comment here that says something like "Tests the order of middleware added by the WhiteLabelTestIntegration" or something like that.

Reply:
Agreed, added comments.